.github/workflows/validate-submodule-pull-request.yml

```diff
@@ -21,8 +21,8 @@ jobs:
     runs-on: ubuntu-22.04
     steps:
       - name: Add Jira Issue Code to Pull Request
-        if: github.event.pull_request != ''
-        uses: Lombiq/GitHub-Actions/.github/actions/add-jira-issue-code-to-pull-request@dev
+        if: github.event.pull_request == opened
+        uses: Lombiq/GitHub-Actions/.github/actions/add-jira-issue-code-to-pull-request@issue/OSOE-505
```

Comment:
This change is not needed.

Reply:
which one ?  the reference @issue/OSOE-505 or the condition of the pull_request event ?

Reply:
If you don't see additional line number which would specify the lines that the comment refers to, then you should consider that it always refers to the line right above the comment. So in this case line 25.

Reply:
done